assets/src/edit-story/components/panels/design/preset/colorPreset/colorPreset.js

```diff
@@ -141,6 +173,11 @@ function ColorPresetPanel({ pushUpdate }) {
                 : __('Apply color preset', 'web-stories')
             }
           >
+            {hasTransparency && !hasGradient && (
+              <OpaqueColorWrapper>
+                <OpaqueColor color={opaqueColor} />
+              </OpaqueColorWrapper>
+            )}
             {isEditMode && <Remove />}
```

Comment:
This has a lot of new conditionals on top of the existing ones - I'd like to see this output covered by tests too.